add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
The code directly uses raw SQL queries and is vulnerable to SQL injection attacks due to the lack of prepared statements. Use prepared statements to prevent SQL injection and consider refactoring to use an object-oriented approach with a Repository pattern for better reuse and testability.